docs/harfbuzz-sections.txt

```diff
@@ -253,6 +253,7 @@ hb_font_get_parent
 hb_font_get_ppem
 hb_font_get_ptem
 hb_font_get_scale
+hb_font_get_track
```

Comment:
What's the Apple API like? I like "tracking" better than "track". Should we predefine LOOSE / NORMAL / etc?
I wish we could combine this with letter-spacing as well.

Reply:
> What's the Apple API like?

I don't know and asked a friend who is more familiar and he had no idea also.

Reply:
Hi; person in question here. As far as Cocoa and Core Text goes, I'm pretty sure all the tracking stuff is handled internally; sorry. You can try finding legacy ATS and ATSUI documentation to see how things are handled there (try http://mirror.informatimago.com/next/developer.apple.com/ for legacy docs)...